### PR TITLE
*: add constraint for binary deployment

### PR DESCRIPTION
**Pull request by lance6716**
### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): some comments from https://github.com/pingcap/dm/issues/1431

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/deploy-a-dm-cluster-using-binary.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Based on this scenario, the following sections describe how to deploy the DM clu
 
 > **Note:**
 >
+> - If you deploy multiple DM-master or DM-worker instances in a single server, ports and working directories of each instances must be unique.
```
Comment (CharLotteiu) [inline en/deploy-a-dm-cluster-using-binary.md:49]:
```suggestion
> - If you deploy multiple DM-master or DM-worker instances in a single server, the port and working directory of each instance must be unique.
```

**Review by CharLotteiu — COMMENTED** [review]
Rest LGTM.

**Review by CharLotteiu — APPROVED** [review]

**Review by GMHDBJD — APPROVED** [review]
/lgtm

**Pull request merged**